packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -181,33 +140,12 @@
   padding: 0 $iui-m;
   overflow-y: auto;
   overflow-y: overlay;
-
-  @at-root .iui-dialog-draggable & {
```

Comment:
Is it not needed?

Reply:
Nope, because I'm setting negative margins in the title bar